src/main/java/uk/gov/ida/eventemitter/EventEmitter.java

```diff
@@ -4,18 +4,24 @@
 
 public class EventEmitter {
 
-    private final boolean sendToRecordingSystem;
+    private final Encrypter encrypter;
+    private final SqsClient sqsClient;
 
     @Inject
-    public EventEmitter(boolean sendToRecordingSystem) {
-        this.sendToRecordingSystem = sendToRecordingSystem;
+    public EventEmitter(final Encrypter encrypter,
+                        final SqsClient sqsClient) {
+        this.encrypter = encrypter;
+        this.sqsClient = sqsClient;
     }
 
-    public void record(Event event) {
-        if (sendToRecordingSystem) {
-            System.out.println(
-                String.format("Event ID: %s, Timestamp: %s, Event Type: %s",
-                    event.getEventId(), event.getTimestamp(), event.getEventType()));
+    public void record(final Event event) {
+        String encryptedEvent = null;
+        try {
+            encryptedEvent = encrypter.encrypt(event);
+            sqsClient.send(event, encryptedEvent);
+        } catch (Exception e) {
+            System.err.println(String.format("Failed to send a message [Event Id: %s] to the queue. Error Message: %s", event.getEventId(), e.getMessage()));
```

Comment:
we have to be a little bit careful here - if the event is null, we might also through an NPE in the catch block, which would be sad!

Reply:
Done.